h2o-r/h2o-package/R/targetencoder.R

```diff
@@ -1,166 +1,50 @@
-#
-# -------------------------- Target Encoding  -------------------------- #
-#
-#' Create Target Encoding Map
+# This file is auto-generated by h2o-3/h2o-bindings/bin/gen_R.py
+# Copyright 2016 H2O.ai;  Apache License Version 2.0 (see LICENSE for details) 
 #'
-#' This is an API for a new target encoding implemented in JAVA.
-#'
-#' Creates a target encoding map based on group-by columns (`x`) and binary target column (`y`).
-#' Computing target encoding for high cardinality categorical columns can improve performance of supervised
-#' learning models.
-#'
-#' @param frame An H2OFrame object with which to create the target encoding map.
-#' @param x List of categorical column names or indices that we want apply target encoding to. 
-#'          Case when item in the list is a list of multiple columns itself is not supported for now.
-#' @param y The name or column index of the response variable in the frame.
-#' @param fold_column (Optional) The name or column index of the fold column in the frame.
-#' @return Returns an object containing the target encoding mapping for each column in `x`.
-#' @seealso \code{\link{h2o.target_encode_transform}} for applying the target encoding mapping to a frame.
+# -------------------------- targetencoder -------------------------- #
+#' @param x A vector containing the \code{character} names of the predictors in the model.
+#' @param blending \code{Logical}. Is blending used ? True by default. Defaults to FALSE.
+#' @param encoded_columns Columnds to encode.
+#' @param target_column Target column for the encoding
+#' @param blending_parameters Parameters used for blending (if enabled). Blending is to be enabled separaterly using the 'blending'
+#'        parameter.
+#' @param data_leakage_handling Data leakage handling strategy. Default to None.
+#' @param model_id Destination id for this model; auto-generated if not specified.
 #' @export
-h2o.target_encode_fit <- function(frame, x, y, fold_column = NULL)
```

Comment:
are we deprecating the old API???

Reply:
I see. It is overridden by the autogenerated changes. It should still be there, only delegate to the new API and mark it as deprecated.